assets/eip-5139/AUTHORS.md

```diff
@@ -0,0 +1,37 @@
+SemVer Authors
+==============
+
+The following people have modified the Semantic Versioning 2.0.0 specification:
+
+ - Tom Preston-Werner
+ - Phil Haack
+ - Haacked
+ - isaacs
+ - Thijs Schreijer
+ - jeffhandley
+ - Alexandr Tovmach
+ - Adam Ralph
+ - Eddie Garmon
+ - Jeff Handley
+ - Krzysztof Piasecki
+ - Doug Beck
+ - Gert de Pagter
+ - Guillermo Calvo
+ - Iulian Onofrei
+ - Ivan Bessarabov
+ - Jo Liss
+ - Johanan Liebermann
+ - Joseph Donahue
+ - Konstantin
+ - Kristian Glass
+ - Mark Amery
+ - OGINO Masanori
+ - Oguz Bilgic
+ - Slipp Douglas
+ - Thomas Schraitle
+ - Tim Vergenz
+ - Todd Reed
+ - Tristram Oaten
+ - Wincent Colaiuta
+ - alexandrtovmach
+ - wolf99
```

Comment:
Not a valid EIP section, and EIPs are CC0 which means attribution is not required (nor appropriate in the standard itself).

Reply:
This is in the assets directory, so AFAIK it doesn't have to be a valid EIP section. The EIP itself is still CC0.

Reply:
Oops, I didn't notice the file break.  My bad.